platforms/minigame/platforms/bytedance/wrapper/engine/VideoPlayer.js

```diff
@@ -41,6 +41,9 @@ if (cc.internal.VideoPlayer) {
     class VideoPlayerImplMiniGame extends cc.internal.VideoPlayerImpl {
         constructor (componenet) {
             super(componenet);
+            this.videoNode = new cc.Node();
+            this.videoNode.addComponent(cc.Sprite);
+            this._node.insertChild(this.videoNode, 0);
```

Comment:
Why not use `addChild` ? Should videoNode have to be the first place in children of `this._node`?

Reply:
Yeah, because it's a video.